ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -294,6 +294,18 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-longduration
     workflow: cucushift-installer-rehearse-aws-ipi
+- as: aws-ipi-longduration-reliability-test-f28
```

Comment:
BTW, if the testing does not care about arch, suggest to move it to arm ci yaml to save our cost.

Reply:
Yes, arch independence. will change.

Reply:
NOTE:
aws and azure support arm from 4.12
gcp support arm from 4.14

BTW, also help move the 4.17 job together.

Reply:
Already done.